plugins/nodes/src/js/pages/NodesOverview.js

```diff
@@ -254,8 +255,14 @@ var NodesOverview = React.createClass({
       });
     const isFiltering = filterExpression && filterExpression.defined;
 
+    const isNodesTableContainer =
+      this.props.children && this.props.children.type === NodesTableContainer;
+
     return (
-      <Page dontScroll>
+      <Page
+        dontScroll={isNodesTableContainer}
+        flushBottom={isNodesTableContainer}
```

Comment:
From the description of the issue it appears to me that we want to `flushBotton` on `dontScroll` pages, curious why distinct those flags?

Reply:
I updated the issue to reflect better the intend we wanted to add the flush bottom specifically to this page which contains the `nodestablecontainer`

Reply:
Am I correct that usually its either `dontScroll` or `flushBottom`? but in this (special) case, we want both. 🤔